Updated the readme.md

CyberFlameGO opened the pull request:
**Updated the readme.**
__*Changes:*__
 - Replaced OS X with macOS because of the OS series name change after El Capitan but also added that some versions of OS X are still compatible
 - Added Chromium to " This tool works with Google Chrome and Opera's built-in U2F implementations"

CyberFlameGO committed:
Updated the readme.md 

Updated the readme.
Changes:
 - Replaced OS X with macOS because of the OS series name change after El Capitan but also added that some versions of OS X are still compatible
 - Added Chromium to " This tool works with Google Chrome and Opera's built-in U2F implementations"

CyberFlameGO committed:
Updated the readme

Firefox 60+ & Safari 13+ natively support WebAuthn (thanks to [oxlay](https://github.com/oxlay) for telling me)

btoews commented on README.md:
```diff
@@ -1,6 +1,6 @@
 ![](https://user-images.githubusercontent.com/1144197/28190263-470a80d2-67e7-11e7-81e6-17895d70bf75.png)
 
-Soft U2F is a software U2F authenticator for OS X. It emulates a hardware U2F HID device and performs cryptographic operations using the OS X Keychain. This tool works with Google Chrome and Opera's built-in U2F implementations as well as with the U2F extensions for [OS X Safari](https://github.com/Safari-FIDO-U2F/Safari-FIDO-U2F) and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/u2f-s ...
+Soft U2F is a software U2F authenticator for macOS and some versions of OS X. It emulates a hardware U2F HID device and performs cryptographic operations using the macOS Keychain. This tool works with Google Chrome/Chromium, Safari, Firefox & Opera's built-in U2F implementations.
```
> macOS and some versions of OS X

I'd rather just say `macOS` than bother with mentioning that some older versions may work.

> Google Chrome/Chromium, Safari, Firefox & Opera's

Please spell out `and` instead of using `&` here.

CyberFlameGO replied:
gotcha will do

CyberFlameGO replied:
Done.

btoews reviewed (commented):
Thanks for opening this!

CyberFlameGO committed:
Fixed according to suggestion

mastayheti suggested "I'd rather just say macOS than bother with mentioning that some older versions may work." and "Please spell out `and` instead of using `&` here.
